src/ripple/crypto/impl/ECIES.cpp

```diff
@@ -97,6 +99,11 @@ static void getECIESSecret (const openssl::ec_key& secretKey, const openssl::ec_
     memset (hbuf, 0, ECIES_KEY_LENGTH);
 }
 
+static void getECIESSecret (uint256 const& secretKey, Blob const& publicKey, ECIES_ENC_KEY_TYPE& enc_key, ECIES_HMAC_KEY_TYPE& hmac_key)
+{
```

Comment:
I actually wasn't aware there was any difference at all - I thought a static function and one in an anon namespace were exactly the same. Huh, learned something new.

I just see more anon namespace in modern C++ code and with rippled making use of all the C++11/14 features it seemed out of place to see "static". It was a style comment.